utils/conda.sh

```diff
@@ -0,0 +1,28 @@
+#!/bin/bash
```

Comment:
Let's convert to using environment.yml files in the same way that sv-tests does.

Reply:
As far as I understand the environment.yml is used to add the packages (e.g. yosys, vtr, etc.). The conda.sh script is used to download conda and add a basic config. I believe this is still needed

Reply:
Yes, conda.sh still needs to exist, it should just use environment.yml as you have done now.